cmake/SkiaUtils.cmake

```diff
@@ -208,6 +210,20 @@ skia_use_system_zlib=false
 skia_use_vulkan=false
 skia_use_zlib=true")
 
+set(SKIA_PRESET_FEATURES_FOR_ANDROID
+"is_official_build=true
+ndk_api=24
+skia_use_system_freetype2=false
+skia_use_system_libpng=false
+skia_use_system_libjpeg_turbo=false
+skia_use_system_harfbuzz=false
+skia_use_system_libwebp=false
+skia_use_expat=true
+skia_use_freetype=true
+skia_use_system_expat=false
+skia_use_system_icu=false
+skia_use_vulkan=true")
```

Comment:
Unless explicitly disabled by setting `skia_use_vulkan=false` as emscripten build does, whether turning on vulkan or not should depend on `VULKAN_VARIABLE` likes other builds.

And this also adds a bit flexibility if we want to lower the api version limit some day.